modules/spring/src/main/java/org/apache/ignite/cache/spring/SpringCache.java

```diff
@@ -74,6 +75,25 @@
     }
 
     /** {@inheritDoc} */
+    @SuppressWarnings("unchecked")
+    @Override public <T> T get(Object key, Callable<T> valueLoader) {
+        Object val = cache.get(key);
+
+        if (val == null) {
+            try {
+                val = valueLoader.call();
+            }
+            catch (Exception ignored) {
+                return null;
```

Comment:
valueLoader.call() - throws an java.lang.Exception if unable to compute a result.